apps/anoma_node/lib/node/transaction/intent_pool.ex

```diff
@@ -116,7 +139,11 @@ defmodule Anoma.Node.Transaction.IntentPool do
       {:ok, :already_present, state}
     else
       Logger.debug("new intent added #{inspect(intent)}")
-      EventBroker.event(Event.new_with_body({:intent_added, intent}), Broker)
+
+      EventBroker.event(
+        Node.Event.new_with_body(state.node_id, {:intent_added, intent}),
```

Comment:
You could just make this Event since you aliased Node.Event

Reply:
? what do you mean?

Reply:
oh, no, we should not do this

let us distinguish between these two separate macros